miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/WorkflowController.java

```diff
@@ -0,0 +1,42 @@
+package uk.ac.bbsrc.tgac.miso.webapp.controller;
+
+import java.io.IOException;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.ModelMap;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.ModelAndView;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import uk.ac.bbsrc.tgac.miso.core.data.workflow.Workflow;
+import uk.ac.bbsrc.tgac.miso.dto.WorkflowStateDto;
+import uk.ac.bbsrc.tgac.miso.service.workflow.WorkflowManager;
+
+/**
+ * Responsible for serving workflow-related JSPs
+ */
+@RequestMapping("/workflow")
+@Controller
+public class WorkflowController {
+  @Autowired
+  WorkflowManager workflowManager;
+
+  @RequestMapping("/new/{workflowName}")
+  public ModelAndView createWorkflow(@PathVariable String workflowName, ModelMap model) throws IOException {
```

Comment:
These 2 endpoints are now accessible, but no links to these pages exist, by intention.